packages/buildpacks/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@heroku-cli/plugin-buildpacks",
+  "version": "0.0.0",
+  "author": "Terence Lee <hone02@gmail.com>",
+  "bugs": "https://github.com/heroku/cli/issues",
+  "dependencies": {
+    "@heroku-cli/color": "^1.1.5",
+    "@heroku-cli/command": "^8.1.15",
+    "@oclif/config": "^1.6.17",
+    "@oclif/plugin-legacy": "^1.0.15",
+    "@types/ansi-styles": "^3.2.0",
+    "@types/lodash": "^4.14.108",
+    "@types/node-fetch": "^2.1.2",
+    "@types/supports-color": "^5.3.0",
+    "buildpack-registry": "^1.0.0",
+    "cli-ux": "^4.2.1",
+    "heroku-cli-util": "^8.0.0",
+    "http-call": "^5.1.2",
+    "lodash": "^4.14.0",
+    "lodash.findindex": "^4.6.0",
+    "node-fetch": "^2.1.2",
```

Comment:
is this necessary?

Reply:
@jdxcode `HTTP` is imported as an argument for `src/buildpack.ts` that i inherited from v5.